modules/integration-node/src/cli.ts

```diff
@@ -75,11 +75,10 @@ const cli = yargs
   .demandCommand()
 
 ;(async (argv) => {
+  // prettier-ignore
   const {
-    _: [command],
-    tolerateFailures,
-    testName,
-    concurrency,
+    // @ts-ignore // ignoring TS2339
+    _: [command], tolerateFailures, testName, concurrency,
```

Comment:
The yargs types update does not fix this?

Reply:
After very careful testing, I can assure you that yargs updates did not fix this. Any ideas? If I have/find any I will try and get rid of this ignore...

Reply:
I will try the change log for `yargs`, maybe there was something interesting...

Reply:
`await`!!!!